.github/workflows/_push_image.yaml

```diff
@@ -0,0 +1,78 @@
+name: Push images
+
+# This is a reusable workflow that is called from any other workflow that needs 
+# to build and push the configurator image to DockerHub
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Version as is going to use for the binary'
```

Comment:
I find the description confusing, what's about: "binary version" and "image tag"